ait/core/server/plugins/openmct.py

```diff
@@ -131,45 +115,27 @@ def _checkConfig(self):
 
 
     def process(self, input_data, topic=None):
-        # msg is going to be a tuple from the ait_packet_handler
-        # (packet_uid, packet)
-        # need to handle log/telem messages differently based on topic
-        # Look for topic in list of stream log and telem stream names first.
-        # If those lists don't exist or topic not in them, try matching text
-        # in topic name.
-
-        processed = False
+        """Process received input message
 
-        if not processed:
-            if "telem_stream" in topic:
-                self.process_telem_msg(input_data)
-                processed = True
+        Received messaged is expected to be a tuple of the form produced
+        by AITPacketHandler.
 
-            elif topic == "log_stream":
-                self.process_log_msg(input_data)
-                processed = True
+        Supported message topics are "telem_stream"
 
-        if not processed:
-            raise ValueError('Topic of received message not recognized as telem or log stream.')
+        """
+        if "telem_stream" in topic:
```

Comment:
Do we want to add telem streams to this plugins config like we did with the GUI plugin? (see [here](https://github.com/NASA-AMMOS/AIT-GUI/blob/1798cfb6971d36edca38ccbcc85eccbf6c3768c4/ait/gui/__init__.py#L256))

Reply:
ya that's a good idea. I dropped the log message handling from the plugin since I don't see it getting used (or being used in the future) but it definitely makes sense to better handle the telemetry streams.